app/helpers/hyrax/iiif_helper.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# OVERRIDE: IIIF Print IiifPrint::IiifHelperDecorator to add #uv_search_param, because it was not being loaded
```

Comment:
Could you add what version (or prob the sha) we're overriding here?

Reply:
Will do!